internal/model/helm.go

```diff
@@ -217,14 +217,14 @@ func HelmTemplateFromStruct(t *testing.T, chart HelmChartBuilder, values HelmVal
 	return decodeK8s(stdErrOut)
 }
 
-func HelmInstallFromStruct(t *testing.T, chart Neo4jHelmChartBuilder, releaseName ReleaseName, values HelmValues) ([]byte, error) {
+func HelmInstallFromStruct(t *testing.T, chart Neo4jHelmChartBuilder, releaseName string, namespace string, values HelmValues) ([]byte, error) {
```

Comment:
any reason for this change ? changing it to string in the function itself is better instead of calling string everytime during the call ?

Reply:
It's just to make it more flexible and explicit. I think the `ReleaseName` structure has too many responsibilities, calling `releaseName.Namespace()` or `releaseName.DiskName()`  doesn't make sense. A good approach would be to use a receiver on this function that holds the namespace and chart.

Reply:
cool